client/app/dedicated/server/rendezvous/dedicated-server-rendezvous.html

```diff
@@ -0,0 +1,30 @@
+<div class="dedicated-server-rendezvous">
+
+    <oui-modal on-dismiss="$ctrl.onDismiss()">
+        <div class="mx-auto">
+            <img src="images/rendezvous.svg"
+                 width="221" height="235px"
+                 class="mx-auto d-block mb-2" />
+            
+            <p class="oui-heading_5 text-center text-justify" data-translate="server_rendezvous_need_help"></p>
```

Comment:
Shouldn't be in the `heading` attribute?

ref: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/modal#api